.github/workflows/script/check-node-modules.sh

```diff
@@ -14,7 +14,7 @@ npm run removeNPMAbsolutePaths
 # Check that repo is still clean
 if [ ! -z "$(git status --porcelain)" ]; then
     # If we get a fail here then the PR needs attention
-    >&2 echo "Failed: node_modules are not up to date. Run 'npm ci' and 'npm run removeNPMAbsolutePaths' to update"
+    >&2 echo "Failed: node_modules are not up to date. Run 'npm ci' and 'npm run removeNPMAbsolutePaths' on a Mac OS X machine to update"
```

Comment:
```suggestion
    >&2 echo "Failed: node_modules are not up to date. Run 'npm ci && npm run removeNPMAbsolutePaths' on a Mac OS X machine to update"
```

Does this need to be on a macos X machine? Technically, BigSur is not X But more importantly, what if you're on linux or windows?s?

Reply:
Unfortunately, if the command is run on Linux or Windows rather than macOS, the dependency `fsevents` (which is Mac-specific) is not installed causing test failues (and likely also issues in production on Mac machines). I agree we should have a way of updating dependencies not on macOS but I'm not entirely sure how we can actually resolve the issue. `npm` isn't really designed for our workflow of checking in dependencies, so as far as I can tell there's no way to make it install a dependency that isn't compatible with the OS you're on.

Reply:
Oh...that sucks. It's not even possible to explicitly add the `fsevents` dependency in the package.json?

Reply:
Sadly not, if we explicitly add the dependency in `package.json` then doing `npm install` on Linux doesn't even work:
```
npm ERR! notsup Unsupported platform for fsevents@2.1.2: wanted {"os":"darwin"} (current: {"os":"linux","arch":"x64"})
```